tests/lax_numpy_test.py

```diff
@@ -3905,5 +3906,49 @@ def f(x):
     check_grads(f, (1.,), order=1)
 
 
+  def testWrappedSignaturesMatch(self):
+    """Test that jax.numpy function signatures match numpy."""
+    # Note: testing signatures for an exact match is problematic, because numpy's signatures
+    # evolve from release to release (mainly adding new parameters when necessary), and jax.numpy
+    # adds additional jax-specific parameters to some functions. Because of this, the test checks
+    # that jax.numpy function parameters are a superset of the wrapped numpy function parameters.
+    # That way, when tested against older numpy versions (where functions often have fewer parameters),
+    # the test should pass, and when run against new numpy releases, the tests will fail indicating
+    # where jax functionality must be upgraded to match.
+
+    # TODO(jakevdp): fix the following signatures.
+    skip = {
+      'allclose', 'amax', 'amin', 'angle', 'argmax', 'argmin', 'around', 'broadcast_to',
```

Comment:
Perhaps it would make sense to have a whitelist of `(function, argument_name)` pairs?

e.g., for some of these we may choose not to implement one or more parameters (e.g., I'm not sure, say, `order` makes sense on `jnp.full` ) but in that case we should not suppress the check entirely.

This list would also be more readable and informative if it included the argument names.

Reply:
I meant this list as a temporary stand-in, that I will work to remove in followup PRs.

My plan is to update jax functions to explicitly raise `NotImplementedError`s on parameters that are deliberately not implemented. In the current state, the docstrings mention those parameters, but the functions don't accept them.

Reply:
I think it's arguably clearer not to include the argument at all in the function signature if `jax.numpy` doesn't support it?

There is even some (possibly buggy) logic to remove unimplemented parameters from jax.numpy docstrings:
https://github.com/google/jax/blob/0f7b6444a796d0b5f4ca19bc4b69a31e731be036/jax/numpy/_util.py#L18

Reply:
Thanks for the input – good points. I'll think about how to make this a more specific and informative list of special-cases, without necessitating adding unimplemented arguments to the `jax.numpy` functions.

Reply:
Done - PTAL